_gsdocs-proposals/2020/proposal_CVMFSPodmanIntegration.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Podman CVMFS integration
+layout: gsdocs_proposal
+project: CVMFS
+year: 2019
+organization:
+ - CERN
+---
+
+## Description
+
+[CernVM-FS (CVMFS)][cvmfs] is a globally-distributed filesystem used to lazily distribute software in different server farm.
+
+[Podman][podman] is an utility to run and use containers.
+It provides the same command line interface than Docker and it run without the need of a privileged daemon.
+These two characteristics make it extremely interesting for workload in scientific data-centers.
+
+It has been shown that only a small portion of all the files in a container images is necessary to run the image itself, this is even more accentuated in scientific images since they usually include big binaries, not always necessary.
```

Comment:
... container image is necessary to run the image itself. This is even more accentuated for scientific ...